cmake/Abseil.cmake

```diff
@@ -0,0 +1,56 @@
+macro(p4c_obtain_abseil)
+  option(
+    P4C_USE_PREINSTALLED_ABSEIL
+    "Look for a preinstalled version of Protobuf in the system instead of installing a prebuilt binary using FetchContent."
+    OFF
+  )
+
+  # If P4C_USE_PREINSTALLED_ABSEIL is ON just try to find a preinstalled version of Abseil.
+  if(P4C_USE_PREINSTALLED_ABSEIL)
+    if(ENABLE_ABSEIL_STATIC)
+      set(SAVED_CMAKE_FIND_LIBRARY_SUFFIXES ${CMAKE_FIND_LIBRARY_SUFFIXES})
+      set(CMAKE_FIND_LIBRARY_SUFFIXES .a)
+    endif()
+    find_package(absl CONFIG)
+    if(NOT absl_FOUND)
+      find_package(absl REQUIRED)
+    endif()
+
+    if(ENABLE_ABSEIL_STATIC)
+      set(CMAKE_FIND_LIBRARY_SUFFIXES ${SAVED_CMAKE_FIND_LIBRARY_SUFFIXES})
+    endif()
+  else()
+    set(P4C_ABSEIL_VERSION "20240116.1")
+    message("Fetching Abseil version ${P4C_ABSEIL_VERSION} for P4C...")
+
+    # Unity builds do not work for Abseil...
```

Comment:
Is this still true?

Reply:
Yes. I had to add this explicitly. There are some global constants